hieradata/site/ls/role/foreman.yaml

```diff
@@ -92,12 +92,36 @@ dhcp::pools:
     range:
       - "139.229.150.112 139.229.150.125"  # ~ /28
     search_domains: "%{alias('dhcp::dnsdomain')}"
-  BTS:
+  BTS_MANKE:
     network: "139.229.151.0"
     mask: "255.255.255.0"
     gateway: "139.229.151.254"
     range:
       - "139.229.151.201 139.229.151.249"  # ~ /27
+  BTS_LSSTCAM:
+    network: "139.229.152.0"
+    mask: "255.255.255.128"
+    gateway: "139.229.152.126"
+    range:
+      - "139.229.152.70 139.229.152.120"  #
```

Comment:
The dangling comment is odd. It would be good to either remove it or add the prefix length to be consistent with the other dhcp pools.

Reply:
you are right! i used an old line and copy/paste it... i will remove it